src/websocket/server.cc

```diff
@@ -257,16 +257,28 @@ future<websocket_parser::consumption_result_t> websocket_parser::operator()(
         }
     }
     if (_state == parsing_state::payload) {
-        if (_payload_length > data.size()) {
-            _payload_length -= data.size();
-            remove_mask(data, data.size());
-            _result = std::move(data);
-            return websocket_parser::stop(buff_t(0));
+        if (data.size() < _remaining_payload_length) {
```

Comment:
A not for ourselves:
the
```patch
-        if (_payload_length > data.size()) {
+       if (data.size() < _remaining_payload_length) {
```
change is idempotent, it changes nothing, but picks up with _payload_length rename to _remaining_payload_length

Correct?

Reply:
Yes.